extensions/ticktick/src/service/osScript.ts

```diff
@@ -15,6 +20,8 @@ const taskObject2Task = (object: Record<string, unknown>): Task => {
     items: object.items as Task["items"],
     kind: object.kind as Task["kind"],
     tags: (object.tags || []) as Task["tags"],
+    startDate: startDate as Task["startDate"],
```

Comment:
format date in render, not store the format date in task object

Reply:
Done, thank you